### PR TITLE
added style guide

### LINKED ISSUE #110: Style guide ba TimorGIS
Objektivu presija kria style guide tamba ita atu mantein style nebee konsistente ba pajina hotu-hotu.

### PR DESCRIPTION
**Pull request by marobo**
Closes #110 

## Description
People said we need a style guide for timorgis.

## Testing
- Temporary I added a link in dropdown settings, click that link to check this works.

### CODE REVIEW

File: map/templates/map/style_guide.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{% load i18n static %}
```
Comment (PeteCoward) [inline map/templates/map/style_guide.html:1]: extend the existing bas site template?

File: map/templates/map/header.html [hunk 2]
```diff
@@ -33,6 +33,9 @@
                                 {% endfor %}
                                 </select>
                             </form>
+                            <a href="{% url 'style_guide' %}">
```
Comment (PeteCoward) [inline map/templates/map/header.html:36]: Do you really want to show the style guide to everyone?

Comment (marobo) [reply map/templates/map/header.html:37]: Solved,
Showing only user is staff

File: map/templates/map/style_guide.html (hunk 1 above)
Comment (marobo) [reply map/templates/map/style_guide.html:2]: Done, already extended it from the base template.

File: map/static/css/timor.css [hunk 3]
```diff
@@ -1,18 +1,144 @@
...
+}
+
+.btn-primary:hover {
+    background-color: var(--blue-1);
+    border-color: var(--blue-2);
+}
+
+.btn-warning {
+    color: var(--white-1);
+    background-color: var(--red-2);
```
Comment (joanico) [inline map/static/css/timor.css:111]: Button color warning karik bele troka ba kor orange.

File: map/static/css/timor.css [hunk 4]
```diff
@@ -1,18 +1,144 @@
...
+
+.btn-warning:hover {
+    color: var(--white-1);
+    background-color: var(--red-1);
+    border-color: var(--red-2);
+}
+.gray-theme {
+    padding: 36px;
+}
+
```
Comment (joanico) [inline map/static/css/timor.css:123]: Bele aumenta tan button Dancer ho kor mean iha nee.

File: map/templates/map/style_guide.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+    <!-- Standard button -->
+    <button type="button" class="btn btn-default">Default</button>
+
+    <!-- Provides extra visual weight and identifies the primary action in a set of buttons -->
+    <button type="button" class="btn btn-primary">Primary</button>
+
+    <!-- Indicates caution should be taken with this action -->
+    <button type="button" class="btn btn-warning">Warning</button>
+
```
Comment (joanico) [inline map/templates/map/style_guide.html:32]: Bele aumenta tan button Dancer iha ne'e.

File: map/static/css/timor.css (hunk 3 above)
Comment (marobo) [reply map/static/css/timor.css:111]: ok diak, hau hanoin hau presija troka naran class nian ba espesifiku butaun nebe ita uza agora dadauk atu ita labele konfunde,... tamba hau hanoin butaun nee ita atu uza ba hamoos deit

File: map/static/css/timor.css (hunk 4 above)
Comment (marobo) [reply map/static/css/timor.css:123]: sim nee mak iha tona leten nee class `btn-warning`

**Comment by marobo** [comment]
Helo @joanico bele review fali.

**Comment by eraulo** [comment]
@joanico ita review ona kolega @marobo nia PR kalae?

File: map/static/css/timor.css (hunk 3 above)
Comment (MarioAP) [reply map/static/css/timor.css:115]: Excelente! Ita bele lao tuir ona style guide nee ona.

**Pull request merged**